cmd/main.go

```diff
@@ -3,14 +3,16 @@ package main
 import (
 	"flag"
 	"io"
-	"log"
```

Comment:
Do you have any conflict with to same "log"

Reply:
I had. resolved